packages/auth/src/platform_browser/strategies/phone.test.ts

```diff
@@ -206,26 +344,53 @@ describe('platform_browser/strategies/phone', () => {
       user = testUser(auth, 'uid', 'email', true);
     });
 
-    it('calls verify phone number', async () => {
-      await reauthenticateWithPhoneNumber(user, '+15105550000', verifier);
+    it('calls verify phone number when recaptcha enterprise is disabled', async () => {
+      if (typeof window === 'undefined') {
+        return;
+      }
+      await reauthenticateWithPhoneNumber(user, '+15105550000', v2Verifier);
```

Comment:
Is this a test number? If so, is it possible someone accidentally delete the number from Firebase console?

Reply:
No, I think this is just a random number (I reused the one that existing unit tests are using). I believe we can use any phone numbers for unit tests; they don't need to be added to Firebase Console.

But, the phone numbers used in **integration tests** are test numbers.